src/main/resources/templates/resource2.html

```diff
@@ -0,0 +1,225 @@
+						</div>
+						<td>vsftpd</td>
+						<td>Backdoor Trojan</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+
+						</td>
+					</tr>
+					<tr>
+						<td>2</td>
+						<td>CVE-2014-6271</td>
+						<td>Bash Code Injection</td>
+						<td>ShellShock</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>3</td>
+						<td>CVE-2014-0160</td>
+						<td>OpenSSL 'Heartbleed'</td>
+						<td>Heartbleed</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>4</td>
+						<td>CVE-2003-0352</td>
+						<td>Buffer Overrun in RPC</td>
+						<td>Microsoft RPC DCOM Interface Overflow</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>5</td>
+						<td>OSVDB-16373</td>
+						<td>gIFTPD dupescan buffer overflow</td>
+						<td>SSH CRC-32 Compensation Attack Detector</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>6</td>
+						<td>OSVDB-60979</td>
+						<td>HTTPd Remote Heap Overflow</td>
+						<td>Null HTTPd Remote Heap Overflow Vulnerability</td>
+						<td>
+							<button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+						</td>
+					</tr>
+					<tr>
+						<td>7</td>
+						<td>EDB-816</td>
+						<td>Anything to PostScript</td>
+						<td>GNU a2ps Local Exploit </td>
+						<td><button type="button" class="btn btn-standard"><span class="btn-special">Realize</span></button>
+							<button type="button" class="btn btn-standard">Save</button>
+
+						</td>
+					</tr>
+
+						</table>
+				</div>
+			</div>
+			<div id="bigdata" class="tab-pane fade">
+			</div>
+			<div id="blockchain" class="tab-pane fade">
+			</div>
+			<div id="mobile" class="tab-pane fade">
+			</div>
+		</div>
+		</div> <!--here-->
+	</div>
+
+
+		<div class="cc-wraper">
+			<div class="cc-contactpop fixed">
+				<h1>Do you need Help? <span>Get in touch.</span></h1>
+				<div class="renseignement">
+					<i class="fa fa-times"></i>
+					<p>We will get back to you as soon as possible. </p>
+				</div>
+				<form>
+					<ul>
+						<li><i class=""></i> <span> Phone: </span>+65 6601 1056</li>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)